modin/pandas/test/utils.py

```diff
@@ -311,13 +314,13 @@
 numeric_agg_funcs = ["sum mean", "sum sum", "sum df sum"]
 
 udf_func = {
-    "return self": lambda df: lambda x, *args, **kwargs: type(x)(x.values),
-    "change index": lambda df: lambda x, *args, **kwargs: pandas.Series(
+    "return self": lambda x, *args, **kwargs: type(x)(x.values),
+    "change index": lambda x, *args, **kwargs: pandas.Series(
         x.values, index=np.arange(-1, len(x.index) - 1)
     ),
-    "return none": lambda df: lambda x, *args, **kwargs: None,
-    "return empty": lambda df: lambda x, *args, **kwargs: pandas.Series(),
-    "access self": lambda df: lambda x, other, *args, **kwargs: pandas.Series(
+    "return none": lambda x, *args, **kwargs: None,
+    "return empty": lambda x, *args, **kwargs: pandas.Series(),
+    "access self": lambda x, other, *args, **kwargs: pandas.Series(
```

Comment:
Any callable that is passed to [`eval_general`](https://github.com/modin-project/modin/blob/fea20642ac30104fe2bc4cd7b616e756f2cfac23/modin/pandas/test/utils.py#L679) as a kwarg is supposed to be a [callable getter for an `operation` argument](https://github.com/modin-project/modin/blob/fea20642ac30104fe2bc4cd7b616e756f2cfac23/modin/pandas/test/utils.py#L720-L722). That had been forcing us to pass a lambda that returns a lambda. Since we [do not pass](https://github.com/modin-project/modin/pull/3746/files#r782026948) UDF functions to the `eval_general` no more, we can get rid of double lambdas here.